src/utils/email/email-api.ts

```diff
@@ -0,0 +1,83 @@
+import axios from 'axios';
+import { Sovryn } from '../../utils/sovryn';
+
+const mailApiKey = process.env.REACT_APP_MAIL_API_KEY;
+const mailSrv = process.env.REACT_APP_MAIL_SRV;
+
+//ADD USER
+export function createUser(newUser): string | void {
+  axios
+    .post(mailSrv + 'addUser', newUser, {
+      headers: {
+        Authorization: mailApiKey,
+      },
+    })
```

Comment:
Looks not safe to me, are we sending api key from frontend to server? It's visible to user right now.

Unless it's public key, API key must be on server side.

```suggestion
//ADD USER
export function createUser(newUser): string | void {
  axios
    .post(mailSrv + 'addUser', newUser)
```